sessions/qc.qmd

```diff
@@ -16,29 +16,46 @@ author:
     affiliation:
       - name: EMBL-EBI
         url: www.ebi.ac.uk
+  - name: Christian Atallah
+    orcid: 0000-0002-4853-4189
+    email: chrisata@ebi.ac.uk
+    affiliation:
+      - name: EMBL-EBI
+        url: www.ebi.ac.uk
 ---
 
 Prerequisites
 -------------
 
 These instructions are for the course VM. To run externally, please refer to the section at the end.
 
-For this tutorial, you'll need to move into the working directory and start a Docker container. Set the variable DATADIR as instructed.
+::: {.callout-note}
+# Keeping your results organised
+This practical (and and the others) will generate quite a few output files from the different commands. It's therefore recommended you keep your results well organised into different subdirectories, starting with the creation of a `qc_results` directory that will contain everything else.
+
+```bash
+mkdir /home/training/qc_results
+```
+:::
+
+For this tutorial, you'll need to move into the working directory and start a Docker container. Set the variables `DATADIR` and `RESDIR` as instructed.
 
 ```bash
 cd /home/training/quality
 chmod -R 777 /home/training/quality
 export DATADIR=/home/training/quality
+chmod -R 777 /home/training/qc_results
+export RESDIR=/home/training/qc_results
 xhost +
 ```
 You will get the message "access control disabled, clients can connect from any host"
 Now start the Docker container:
 
 ```bash
-docker run --rm -it  -e DISPLAY=$DISPLAY  -v $DATADIR:/opt/data -v /tmp/.X11-unix:/tmp/.X11-unix:rw -e DISPLAY=unix$DISPLAY microbiomeinformatics/biata-qc-assembly:v2021
+docker run --rm -it  -e DISPLAY=$DISPLAY  -v $DATADIR:/opt/data -v $RESDIR:/results -v /tmp/.X11-unix:/tmp/.X11-unix:rw -e DISPLAY=unix$DISPLAY microbiomeinformatics/biata-qc-assembly:v2021
```

Comment:
Any particular reason to use `/results` instead of `/opt/results` (for consistency with `/opt/data`) ?

Reply:
nope no reason, but I do like the consistency, I'll change it up thanks 👍